homework/index.js

```diff
@@ -30,13 +30,188 @@
     return elem;
   }
 
+  const root = document.getElementById('root');
+  const container = createAndAppend('div', root, { id: 'container' });
+  function getSelectedData(value, data) {
+    let selected;
+    data.forEach(item => {
+      if (item.name === value) {
+        selected = item;
+      }
+    });
+    return selected;
+  }
+
+  function generateSelections(repos) {
+    const header = createAndAppend('header', root, {});
+    const selectionElem = createAndAppend('select', header, {
+      id: 'repositories',
+    });
+    let id = 0;
+
+    repos.forEach(repo => {
+      createAndAppend('option', selectionElem, {
+        id: id++,
+        value: repo,
+        text: repo,
+      });
+    });
+  }
+
+  function generateInfoSection(selected, data) {
+    const div = createAndAppend('div', container, { id: 'infoDiv' });
+    const selectedData = getSelectedData(selected, data);
```

Comment:
Tip: look into the function `Array#find`. It does sort of what you want here :D.